### PR TITLE
Office users list via React Admin

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

System administrators need to be able to log in and see office user data.

## Setup

1. `make server_run`
2. `make client_run`
3. Visit `adminlocal:3000` and log in
4. You should be taken to a screen that has a list of all office users

## Code Review Verification Steps

* [x] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [x] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/164884956) for this change

## Screenshots
Screen Recording 2019-07-30 at 10 03 01

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,12 +4,15 @@ import 'core-js/stable';
...
 import 'uswds';
 
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 import './index.css';
-import '../node_modules/uswds/dist/css/uswds.css';
 
+if (!isSystemAdminSite) {
+  require('../node_modules/uswds/dist/css/uswds.css');
+}
```
Comment (garrettqmartin8) [inline src/index.js]: USWDS messes with React Admin's styles, so we're conditionally loading that CSS here.

File: src/App/index.jsx [hunk 2]
```diff
@@ -61,14 +67,13 @@ const App = () => {
...
     );
-  else if (isAdminSite)
+  else if (isSystemAdminSite)
     return (
-      <Provider store={store}>
-        <AppContext.Provider value={adminContext}>
-          <Admin />
-        </AppContext.Provider>
-      </Provider>
+      <AppContext.Provider value={adminContext}>
```
Comment (garrettqmartin8) [inline src/App/index.jsx]: Everything seems to play more nicely when we don't wrap React Admin in our own Redux provider.

**Comment by mikena-truss** [comment]
Tagging @LeDeep since he put in the first changes for this.

**Comment by donaldthai** [comment]
@garrettqmartin8 looks like the Edge browser has a problem with the query request: 
image

**Comment by garrettqmartin8** [comment]
@donaldthai hmmm. I tested with both IE and Edge without any hiccups. Would love to pair on this and see what it takes to reproduce.

**Comment by donaldthai** [comment]
The admin http client wasn't sending cookies in the request for MS Edge. We had to set the request to include "credentials", `request.credentials = 'same-origin'`, to send the cookies.

File: src/scenes/SystemAdmin/Menu.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const Menu = props => {
+  const resources = props.resources;
+  return (
+    <div>
+      {resources.map(resource => (
+        <MenuItemLink
+          key={resource.name}
+          to={`/${resource.name}`}
+          primaryText={(resource.options && resource.options.label) || resource.name}
+          onClick={() => console.log('hi')}
```
Comment (mikena-truss) [inline src/scenes/SystemAdmin/Menu.jsx]: Can this `console.log` be removed?

File: src/scenes/SystemAdmin/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import { history } from 'shared/store';
+import React from 'react';
+import Menu from './Menu';
+
+const dataProvider = restProvider('/admin/v1');
+
+const AdminLayout = props => <Layout {...props} menu={Menu} />;
+const AdminPagination = props => <Pagination rowsPerPageOptions={[]} {...props} />;
+const UserList = props => (
+  <List {...props} pagination={<AdminPagination />} perPage={500}>
```
[the remark this answers is not in the document]
Comment (garrettqmartin8) [reply src/scenes/SystemAdmin/Home.jsx]: Yeah - more to come on that very soon!

**Comment by garrettqmartin8** [comment]
@mikena-truss made some adjustments based on your feedback.

**Comment by lynzt** [comment]
disregard my comment above... I was on master...

**Pull request merged**